webserver.py

```diff
@@ -91,30 +91,37 @@ def save(self, slug):
                                [slug, json_game_state, json_game_state])
 
     def initialize(self):
-        for seed_id in GAME_CONFIG['seeds']:
-            if self.data['seedCounts'].get(seed_id) is None:
-                self.data['seedCounts'][seed_id] = 0
         for resource in GAME_CONFIG['starting_resources']:
             if self.data['resources'].get(resource) is None:
-                self.data['resources'][resource] = GAME_CONFIG['starting_resources'][resource]
+                if GAME_CONFIG['starting_resources'][resource] > 0:
+                    self.data['resources'][resource] = GAME_CONFIG['starting_resources'][resource]
+        if self.data.get('plots') is None:  # temp
```

Comment:
I like marking the parts of code that are just temporary.  Making it clear where they start and end and why they're temporary will make it easier to clean up later.